File: src/main/java/models/MovieDetails.java
package models;

import com.j256.ormlite.field.DatabaseField;
import com.j256.ormlite.table.DatabaseTable;

/**
 * Created by martin on 21/04/17.
 */
@DatabaseTable(tableName = "moviedetails")

public class MovieDetails {

    @DatabaseField
    private int idMovie;
    @DatabaseField //title
    private String c00;
    @DatabaseField //description
    private String c01;
    @DatabaseField //plot
    private String c03;
    @DatabaseField //Guion
    private String c06;
    @DatabaseField //poster
    private String c08;
    @DatabaseField //genero
    private String c14;
    @DatabaseField //director
    private String c15;
    @DatabaseField //estudio
    private String c18;
    @DatabaseField //triler
    private String c19;
    @DatabaseField //imdb
    private String uniqueid_value;
    @DatabaseField
    private String premiered;
    @DatabaseField
    private String rating;
    @DatabaseField
    private String votes;
    @DatabaseField //pais
    private String c21;
    @DatabaseField //fanart
    private String c20;


    MovieDetails(){

    }

}

File: src/main/app/models/MovieDetails.kt
package models

import com.j256.ormlite.field.DatabaseField
import com.j256.ormlite.table.DatabaseTable

/**
 * Created by martin on 21/04/17.
 */
@DatabaseTable(tableName = "moviedetails")
class MovieDetails internal constructor() {

    @DatabaseField
    private val idMovie: Int = 0
    @DatabaseField //title
    private val c00: String? = null
    @DatabaseField //description
    private val c01: String? = null
    @DatabaseField //plot
    private val c03: String? = null
    @DatabaseField //Guion
    private val c06: String? = null
    @DatabaseField //poster
    private val c08: String? = null
    @DatabaseField //genero
    private val c14: String? = null
    @DatabaseField //director
    private val c15: String? = null
    @DatabaseField //estudio
    private val c18: String? = null
    @DatabaseField //triler
    private val c19: String? = null
    @DatabaseField //imdb
    private val uniqueid_value: String? = null
    @DatabaseField
    private val premiered: String? = null
    @DatabaseField
    private val rating: String? = null
    @DatabaseField
    private val votes: String? = null
    @DatabaseField //pais
    private val c21: String? = null
    @DatabaseField //fanart
    private val c20: String? = null

}
